src/routes/exercise-finished/ExerciseFinishedScreen.tsx

```diff
@@ -146,6 +178,32 @@ const ExerciseFinishedScreen = ({ navigation, route }: ExerciseFinishedScreenPro
       backgroundColor={unlockedNextExercise ? theme.colors.correct : theme.colors.primary}
       lightStatusBarContent={!unlockedNextExercise}
       bottomBackgroundColor={theme.colors.background}>
+      {exercise === EXERCISES.length - 1 ? (
+        <ModalSkeleton visible={isModalVisible} onClose={() => setIsModalVisible(false)}>
+          <ModalContainer>
+            <LightBulbIconBlack
+              width={theme.spacingsPlain.xxl}
+              height={theme.spacingsPlain.xxl}
+              testID='light-bulb-icon-black'
+            />
+            <HeaderText>{getLabels().repetition.hintModalHeaderText}</HeaderText>
+            <ContainerText>{getLabels().repetition.hintModalContentText}</ContainerText>
+            <FooterImageIcon />
+            <HintButtonWrapper>
+              <HintButton
+                onPress={() => setIsModalVisible(false)}
+                label={getLabels().repetition.repeatNow}
+                buttonTheme={BUTTONS_THEME.contained}
+              />
+              <HintButton
+                onPress={() => setIsModalVisible(false)}
```

Comment:
When clicking on "repeatLater" the app should navigate to the long term exercise.

Reply:
@SteffiStuffel `Repetition` Navigator.Screen was commented out, so I left it for now without including it. Maybe I should add here a comment to include it later when everything will be ready

Reply:
Yes this sounds good, please also add this as a comment to the clean-up ticket, so we will not miss it.

Reply:
@SteffiStuffel seems like the repetition navigator screen has been uncommented. I tried to add navigation to long term exercise, but seems like there is an error.

Reply:
@lunars97 Not sure what the problem is here, feel free to explain a little more in detail or we can have a look together at it tomorrow.

Reply:
@SteffiStuffel sure. I want to navigate on `Jetzt wiederholen` to Repetition screen. But it is causing error on onPress saying  `The action 'NAVIGATE' with payload {"name":"Repetition"} was not handled by any navigator.
Do you have a screen named 'Repetition'?`